vpr/src/base/read_options.cpp

```diff
@@ -1650,6 +1650,11 @@ argparse::ArgumentParser create_arg_parser(std::string prog_name, t_options& arg
         .default_value("off")
         .show_in(argparse::ShowIn::HELP_ONLY);
 
+    route_timing_grp.add_argument<bool, ParseOnOff>(args.read_edge_metadata, "--read_edge_metadata")
+        .help("Read edge metadata from --read_rr_graph.  This is typically not required, and therefore defaults to false.")
```

Comment:
When is it required?

The format used elsewhere would be "This is typically not required. (Default: off)"

Reply:
> When is it required?

If you care about the edge metadata?  Examples would be round trips through VPR, profiling metadata logic, etc.  The VPR core never touches the data, so it is cheaper to just not read it.